### PR TITLE
Use print() function in both Python 2 and Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Legacy __print__ statements are syntax errors in Python 3 but __print()__ function works as expected in both Python 2 and Python 3.

### CODE REVIEW

**Comment by scottleibrand** [comment]
Do we need to do something else to enable Travis on this repo?

**Comment by cclauss** [comment]
@openaps needs to visit https://travis-ci.org/openaps and flip the repo switch on or run the first test.

**Comment by scottleibrand** [comment]
Enabled Travis. Closing and reopening the PR to force a build.

File: dexcom_reader/database_records.py [hunk 1]
```diff
@@ -26,7 +31,7 @@ def _ClassSize(cls):
   @property
   def FMT(self):
     self._CheckFormat()
-    return _ClassFormat()
+    return self._ClassFormat()
```
Comment (cclauss) [inline dexcom_reader/database_records.py]: What is the desired functionality here?  It it to call __self._CheckFormat()__ or is it to return the bound method __self._CheckFormat__ without calling it or both?!?

**Comment by cclauss** [comment]
It passes the tests but brings up a new question about the desired functionality of the property __BaseDatabaseRecord.FMT__

**Comment by cclauss** [comment]
Please review the change to __FMT__ at https://github.com/openaps/dexcom_reader/pull/22/files/4c75b48b70ca61f4111bcf5315c7c4527ba7a85c#diff-1dcdd31fd116bbbdee73d915e7b2023aR31

**Comment by bewest** [comment]
Need to target `dev` branch as the standard workflow.  See https://github.com/openaps/stdprojectexample and "git flow" for more details.  Travis is enabled for this project.

**Comment by cclauss** [comment]
The __dev__ branch is 2 commits ahead, 3 commits behind __master__.

**Comment by cclauss** [comment]
Closing in favor of #23 which is done on the __dev__ branch, not the __master__ branch.

**Pull request closed without merging**